features/step_definitions/cms/email_template_steps.rb

```diff
@@ -27,6 +27,5 @@
   t = CMS::EmailTemplate.dup.extend(CMS::EmailTemplate::ProviderAssociationExtension)
 
   content = t.find_default_by_name(name).published
-
-  page.find :xpath, XPath.descendant(:textarea)[XPath.text.is(content)]
+  page.find :xpath, XPath.descendant(:textarea)[XPath.text.is(content)], visible: :hidden
```

Comment:
Similarly to https://github.com/3scale/porta/pull/1143/files#r321299524, Josemi explained that it is because the editor to do the templates is a JS library that uses a hidden textarea (like in background) that is used to send the form.